app/services/data_stage/process_school_changes.rb

```diff
@@ -0,0 +1,90 @@
+# frozen_string_literal: true
+
+module DataStage
+  class ProcessSchoolChanges < ::BaseService
+    include GiasTypes
+
+    UNHANDLED_ATTRIBUTES = %w[
+      administrative_district_code
+      administrative_district_name
+      school_type_code
+      school_type_name
+      section_41_approved
+      ukprn
+    ].freeze
+
+    def call
+      DataStage::SchoolChange.unhandled.status_changed.find_each do |change|
+        next if has_unhandled_attributes?(change.attribute_changes)
+
+        if change.attribute_changes.key? "school_status_code"
+          handle_status_code_change(change)
+        end
```

Comment:
Should we handle status code change before we skip?

Reply:
Not at the moment as the status code change is the only change we want to handle, we don't want to do that if there are other major changes too (don't understand what's needed for those yet) because we'd then mark the change as being handled. I've got some discussions with the business about what we should do planned.